lib/beacon/loader/layout_module_loader.ex

```diff
@@ -28,6 +29,7 @@ defmodule Beacon.Loader.LayoutModuleLoader do
 
   defp render_layout(%Layout{} = layout) do
     Beacon.Util.safe_code_heex_check!(layout.body)
+    runtime_css = RuntimeCSS.compile!(layout)
```

Comment:
We need to call `render_layout` for the layout of a page whenever a page is updated.

Reply:
Ahh yes, I didn't see it in this pr and forgot we were doing a "load the whole world" on publish_page. I added #76 to make that more focused.